owasp-dependency-check-suppressions.xml

```diff
@@ -806,4 +806,13 @@
     <!-- seems not applicable to plugin -->
     <cve>CVE-2022-45048</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+     file name: hadoop-client-runtime-3.3.5.jar
+     ]]></notes>
+    <!-- this one is windows only? https://nvd.nist.gov/vuln/detail/CVE-2022-26612 -->
+    <cve>CVE-2022-26612</cve>
+    <!-- this one seems to apply to backend server? https://nvd.nist.gov/vuln/detail/CVE-2023-25613 -->
```

Comment:
This one is for Apache Kerby. Can we instead exclude it (if we're not using it) or upgrade it to 2.0.3 (if we are using it transitively)?

Reply:
it's shaded inside the Hadoop client jar, so probably not?

Reply:
Could we upgrade the hadoop client?

If we can't do any of this stuff, then I agree this appears to be about the backend server, which presumably we wouldn't be using in a client. So I think a suppression is OK. But it should be a last resort due to the severity. (Even if it's a false positive, suppressed high-sev false positives creates a bunch of work for downstream consumers that run their own scans.)

Ditch the "?" in the comment, though, once this analysis is complete. We need to be confident when suppressing things.

Reply:
we are on the newest version unfortunately, 3.3.5 https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-client-runtime, so not an option either.

Reply:
>Ditch the "?" in the comment, though, once this analysis is complete. We need to be confident when suppressing things.

heh, i'm not sure i'm ever fully confident when suppressing things, this stuff stresses me out